keps/sig-api-machinery/00xx-admission-webhooks-to-ga.md

```diff
@@ -314,10 +352,13 @@ package v1
 type ScopeType string
 
 const (
-     // ClusterScope means that scope is limited to cluster objects.
+     // ClusterScope means that scope is limited to cluster-scoped objects.
+     // Namespace API objects are considered cluster-scoped.
```

Comment:
Somehow, there's like 6 copies of this comment in the KEP. That seems like more than is a good idea.

Reply:
reduced to 4 instances :)